.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Renamed so that it can be automatically parsed via `require` in the karma config.